### PR TITLE
added hyperlinks to integration cards

### LINKED ISSUE #1775: Add documentation hyperlink to each integration card
### Current Behavior
The cards displayed in the visualization on https://meshery.io/integrations are not clickable.

### Desired Behavior
Programmatically ensure that each card has a hyperlink to its respective Meshery Docs page.

Example:

Camel K card should link to https://docs.meshery.io/extensibility/integrations/camel-k.

---
<h4>Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources</h4>

- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🖥 [Contributing to Meshery Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by hargunkaur286**
**Description**

This PR fixes #1775

**Notes for Reviewers**
@leecalcote @sudhanshutech Can you please review this?

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by sudhanshutech** [comment]
@hargunkaur286 lets discuss in website call today. Add as an agenda please if you would

**Comment by iArchitSharma** [comment]
@sudhanshutech is the `../integrations/data.js` coming from spreadsheet?

**Comment by hargunkaur286** [comment]
> @hargunkaur286 some of the permalinks are wrong

They are coming from data.js file, should I make changes in that file?

File: integrations/index.html [hunk 1]
```diff
@@ -256,6 +256,7 @@ <h2 class="integration-subhead">
           const element = document.createElement("div");
           element.className = "element";
           element.style.backgroundColor = "rgba(255, 255, 255," + "1" + ")";
+          element.onclick = () => window.open(table[i].permalink, "_blank");
```
Comment (leecalcote) [inline integrations/index.html:259]:
```suggestion
          element.onclick = () => window.open(table[i].permalink, "_self");
```

**Review by leecalcote — APPROVED** [review]
This is great. Thank you, @hargunkaur286 👏

**Comment by leecalcote** [comment]
@iArchitSharma raise a new issue, please.

**Pull request merged**